src/components/StyledButton.js

```diff
@@ -67,6 +68,7 @@ const StyledButtonContent = styled(tag.button)`
   ${textAlign}
   ${width}
   ${height}
+  ${margin}
```

Comment:
This is not required as we already add `space` which provides a margin prop with `m`, `mx`...etc

See https://styled-system.com/api/#space